src/OVAL/probes/unix/process58-devname.h

```diff
@@ -1,12 +0,0 @@
-
```

Comment:
src/OVAL/probes/unix/process58_probe.c and src/OVAL/probes/unix/process_probe.c use this file